lib/sequenceserver/routes.rb

```diff
@@ -99,13 +99,19 @@ class Routes < Sinatra::Base
       database_ids = params[:database_ids].split(/\s/)
 
       sequences = Sequence::Retriever.new(sequence_ids, database_ids,
-                                          params[:download])
+                                          true)
```

Comment:
The third parameter (`true`/`false`) to `Sequence::Retriever.new` indicates whether the sequences retrieved from blast databases should be written to a file (`true`) or not (`false`). Default is `false`.

While passing `true` here will work. The code is mis-leading and does additional work that's not required.